README.md

```diff
@@ -120,39 +120,32 @@ Intent is any action takes place on fragment. It may or may not be user initiate
 ### Components
 Components are reusable parts of UI. Components can be used in any fragment using the `by composable` delegate provided by the `BaseFragment`.
 
-Components can be anything from a simple component to hide and show the loading indicator to a complex component like `ListComponent` that can manage normal and nested recycler view effciently.
+Components can be anything from a simple component to hide and show the loading indicator to a complex component like `ListComponent` that can manage normal and nested recycler view efficiently.
 
 ### Fragment 
 Each Fragment must extend the `BaseFragment`. 
 The `BaseFragment` provides the [`ViewModel`](#view-model) with the navigator and the [`Screen`](#screen). It listens to the screen state and effect live data from the view model and notifies the fragment about it. It also binds and unbinds all the components in the appropriate lifecycle callbacks.
 
-Each Fragment may receive the [`Screen`](#screen) as aguments when navigating.
+Each Fragment may receive the [`Screen`](#screen) as arguments when navigating.
 
 ### Screen
 A [`Screen`](presentation-entity/src/main/java/com/wednesday/template/presentation/screen/Screen.kt) is a class that represents a `Fragment` in the context of navigation. It holds the `path or id` used by the navigator to navigate to a `Fragment` and also holds any arguments required to navigate to that `Fragment`.
 
 ## Flavors
-The template comes with built-in support for 3 flavors. Each flavor uses a diffrent `main.dart` file.
-- Dev - [`main_dev.dart`](lib/entrypoints/main_dev.dart)
-- QA - [`main_qa.dart`](lib/entrypoints/main_qa.dart)
-- Prod - [`main_prod.dart`](lib/entrypoints/main_prod.dart)
+The template comes with built-in support for 3 flavors
+- Dev - This flavour is used across the development stage 
+- QA - This flavour would be used when the app is under review by the QA Team or the Project Manager
+- Prod - This flavour would be used when the app is ready to be deployed to a store or shipped to the user
 
-You can setup any environment specific values in the respective `main.dart` files.
-
-To run a specific falvor you need to specify the flavor and target file.
-```shell
- flutter run --flavor qa -t lib/entrypoints/main_qa.dart
-```
-
-**To avoid specifying all the flags every time, use the [`run.sh`](scripts/README.md#run) script**
+| Note: Flavour specific configurations can be made in the [app.gradle](app/app.gradle) file under the `buildTypes` block
 
 Read the [scripts documentation](scripts/README.md) to learn about all the scrips used in the project.
```

Comment:
I think we can remove this line since `Flavors` block is not referencing any scripts.

Reply:
I don't get it, can you specify what exactly you want changed?